modules/combined/examples/periodic_strain/global_strain_pfm.i

```diff
@@ -0,0 +1,364 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 50
+  ny = 50
+  xmin = -0.5
+  xmax = 0.5
+  ymin = -0.5
+  ymax = 0.5
+[]
+
+[MeshModifiers]
+  [./cnode]
+    type = AddExtraNodeset
+    coord = '0.0 0.0'
+    new_boundary = 100
+  [../]
+  [./anode]
```

Comment:
I think `anode` is unnecessary as the global tensor does not allow for rotation and neither do the periodic displacements.